src/server/game/Maps/Map.h

```diff
@@ -938,6 +938,8 @@ class TC_GAME_API InstanceMap : public Map
         bool HasPermBoundPlayers() const;
         uint32 GetMaxPlayers() const;
         uint32 GetMaxResetDelay() const;
+        TeamId GetPlayersTeamId() const { return i_script_team; }
```

Comment:
Name it `GetTeamInInstance` so it matches the script member please.

PS: Just to be safe, does this work with cross-faction groups?

Reply:
rename rdy. About cross-faction : instance will be created by first member which will enter instance. 
For example if your cross-faction group consist of 1 horde and 19 alliance -> and this 1 horde will enter in instance first -> instance will have TeamId = TEAM_HORDE